lib/modules/apostrophe-schemas/index.js

```diff
@@ -1437,6 +1443,9 @@ module.exports = {
       converters: {
         string: function(req, data, name, object, field, callback) {
           object[name] = self.apos.launder.float(data[name], field.def, field.min, field.max);
+          if (_.isEmpty(data[name])) {
```

Comment:
Here I don't have that concern. If the field has an empty value, then it will now be set to null, so it does not get a 0 into the database. The current implementation was to launder and overwrite the values server side, if a value is outside the allowed min / max which happens before in the `launder` function, but the will only happen if client side validation fails (I guess it can be possible to set the configuration in a combo where client side validation does not have effect + browsers/bots that does not accept max length on the field etc.). The min / max validation *when* there's a value has been improved in user.js.